lib/c-api/tests/wasmer_wasm.h

```diff
@@ -0,0 +1,26 @@
+// This header file is used only for test purposes! It is used by unit
+// test inside the `src/` directory for the moment.
+
+#if !defined(TEST_WASMER_WASM)
+
+#define TEST_WASMER_WASM
+
+#include <string.h>
+#include <stdio.h>
+#include "../wasmer_wasm.h"
+
+// Assert that a `wasm_name_t` equals something.
+void wasmer_assert_name(const wasm_name_t* name, const char* expected) {
+  assert(name->size == strlen(expected) && strncmp(name->data, expected, name->size) == 0);
+}
+
+
+// Helper to quickly create a `wasm_byte_vec_t` from a string, à la
+// `wasm_name_new_from_string`.
+static inline void wasmer_byte_vec_new_from_string(
+  wasm_byte_vec_t* out, const char* s
+) {
```

Comment:
Consider running `clang-format -i filename` over this file. (`-i` is in-place editing.)

Reply:
Done, thanks.